src/project.ts

```diff
@@ -32,6 +33,14 @@ export interface ProjectOptions {
    * @default "."
    */
   readonly outdir?: string;
+
+  /**
+   * Whether to enable `gitPod`
+   * It can be customized later if needed
+   *
+   * @default false
+   */
+  readonly gitPod?: boolean;
```

Comment:
I think this should be `gitpod`, along with the other occurrences in the file
```suggestion
  readonly gitpod?: boolean;
```

Reply:
Formally its Gitpod so I guess that makes it one word in which case gitpod would be correct.

Reply:
renamed